'dependencyManagement' Errors during the Analyzer Step

Am I getting the errors below because those JARs are flat dir repositories?

**Error 1:**
```bash
Caused by: ModelBuildingException: 2 problems were encountered while building the effective model for org.glassfish.jaxb:jaxb-core:2.2.11
[WARNING] Reporting configuration should be done in <reporting> section, not in maven-site-plugin <configuration> as reportPlugins parameter. @ 
[ERROR] 'dependencyManagement.dependencies.dependency.systemPath' for com.sun:tools:jar must specify an absolute path but is ${tools.jar} @ com.sun.xml.bind.mvn:jaxb-parent:2.2.11, /root/.gradle/caches/modules-2/files-2.1/com.sun.xml.bind.mvn/jaxb-parent/2.2.11/9496f6747e6d5ab5da869941dea4143969122b91/jaxb-parent-2.2.11.pom, line 188, column 29
```
**Error 2:**
```bash
11:04:20.952 [analyzer-1] WARN  com.here.ort.analyzer.managers.utils.MavenLogger - Failed to build parent project for com.sun.xml.bind.mvn:jaxb-txw-parent:pom:2.2.11
org.apache.maven.project.ProjectBuildingException: Some problems were encountered while processing the POMs:
[WARNING] Reporting configuration should be done in <reporting> section, not in maven-site-plugin <configuration> as reportPlugins parameter. @ line 497, column 32
[ERROR] 'dependencyManagement.dependencies.dependency.systemPath' for com.sun:tools:jar must specify an absolute path but is ${tools.jar} @ line 188, column 29

        at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:207) ~[maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:342) ~[maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:299) ~[maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.initParent(DefaultProjectBuilder.java:960) [maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.initProject(DefaultProjectBuilder.java:680) [maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:188) [maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:342) [maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:299) [maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.initParent(DefaultProjectBuilder.java:960) [maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.initProject(DefaultProjectBuilder.java:680) [maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:188) [maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:342) [maven-core-3.6.3.jar:3.6.3]
        at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:299) [maven-core-3.6.3.jar:3.6.3]
        at com.here.ort.analyzer.managers.utils.MavenSupport$parsePackage$$inlined$let$lambda$1.invoke(MavenSupport.kt:517) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.utils.MavenSupport$parsePackage$$inlined$let$lambda$1.invoke(MavenSupport.kt:95) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.utils.MavenSupport.wrapMavenSession(MavenSupport.kt:608) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.utils.MavenSupport.parsePackage(MavenSupport.kt:516) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.utils.MavenSupport.parsePackage$default(MavenSupport.kt:494) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle.parseDependency(Gradle.kt:265) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle.parseDependency(Gradle.kt:287) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle.parseDependency(Gradle.kt:287) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle.parseDependency(Gradle.kt:287) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle.parseDependency(Gradle.kt:287) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle.parseDependency(Gradle.kt:287) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle.parseDependency(Gradle.kt:287) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle.access$parseDependency(Gradle.kt:67) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle$resolveDependencies$4.invoke(Gradle.kt:192) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle$resolveDependencies$4.invoke(Gradle.kt:67) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.utils.UtilsKt.temporaryProperties(Utils.kt:304) [utils-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.managers.Gradle.resolveDependencies(Gradle.kt:165) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.PackageManager.resolveDependencies(PackageManager.kt:213) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at com.here.ort.analyzer.Analyzer$analyzeInParallel$$inlined$use$lambda$1$1$1.invokeSuspend(Analyzer.kt:143) [analyzer-0.1.0-SNAPSHOT.jar:?]
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [kotlin-stdlib-1.3.61.jar:1.3.61-release-180 (1.3.61)]
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56) [kotlinx-coroutines-core-1.3.3.jar:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
        at java.lang.Thread.run(Thread.java:835) [?:?]

```

--- a/analyzer/src/main/kotlin/managers/Maven.kt
+++ b/analyzer/src/main/kotlin/managers/Maven.kt
@@ -175,7 +175,8 @@ class Maven(
 
         try {
             val dependencies = node.children.mapNotNull { child ->
-                if (child.artifact.identifier().startsWith("jdk.tools:jdk.tools:")) {
+                val toolsJarCoordinates = listOf("com.sun:tools:", "jdk.tools:jdk.tools:")
+                if (toolsJarCoordinates.any { child.artifact.identifier().startsWith(it) }) {
                     log.info { "Omitting the Java < 1.9 system dependency on 'tools.jar'." }
                     null
                 } else {

